test/test_mps.py

```diff
@@ -2745,6 +2745,82 @@ def run_lu_factor_ex_test(size, *batch_dims, check_errors):
         run_lu_factor_ex_test(32, 10, 10, check_errors=False)
         run_lu_factor_ex_test(32, 2, 2, 2, 2, 10, 10, check_errors=True)
 
+    def test_linalg_solve(self):
```

Comment:
Just curious, is this test really necessary, as OpInfo tests presumably already covers that?